contracts/contracts/Adjudicator.sol

```diff
@@ -6,8 +6,8 @@ import "./lib/ReEncryptionValidator.sol";
 import "./lib/SignatureVerifier.sol";
 import "./IStakingEscrow.sol";
 import "./proxy/Upgradeable.sol";
-import "../zeppelin/math/SafeMath.sol";
-import "../zeppelin/math/Math.sol";
+import "./lib/AdditionalMath.sol";
```

Comment:
Probably we don't need AdditionalMath here too

Reply:
removed